index.js

```diff
@@ -798,6 +818,8 @@ function encRange (e, opts) {
   if (opts.gte !== undefined) opts.gte = enc(e, opts.gte)
   if (opts.lt !== undefined) opts.lt = enc(e, opts.lt)
   if (opts.lte !== undefined) opts.lte = enc(e, opts.lte)
+  if (opts.sub && !opts.gt && !opts.gte) opts.gt = enc(e, Buffer.alloc(1))
+  if (opts.sub && !opts.lt && !opts.lte) opts.lt = enc(e, Buffer.alloc(1).fill(255))
```

Comment:
Buffer.from([255]) and move it to a static field like your SEP, same for the one above

Reply:
Done